homeassistant/components/tibber/strings.json

```diff
@@ -101,5 +101,23 @@
         "description": "Enter your access token from {url}"
       }
     }
+  },
+  "exceptions": {
+    "send_message_timeout": {
+      "message": "Timeout sending message with Tibber"
+    }
+  },
+  "issues": {
+    "migrate_notify": {
+      "title": "Migration of Tibber notify service",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "description": "The Tibber `notify` service has been migrated. A new `notify` entity is available now.\n\nUpdate any automations to use the new `notify.send_message` service exposed by this new entity. When this is done, fix this issue and restart Home Assistant.",
```

Comment:
There should be no need to restart I think?

Reply:
This whole part needs to be taken out since we use the helper now.

Reply:
Was wondering if that was the case but was thinking perhaps the repairs are so specific but good to know 👍

Reply:
Removed now